.github/workflows/build-test.yml

```diff
@@ -80,9 +79,12 @@ jobs:
         run: |
           export RUST_BACKTRACE="full"
           cargo install ic-wasm
-          cargo install icx-proxy
+          cargo install icx-proxy --version 0.10.1
```

Comment:
why do we need a fixed version?

Reply:
We've never fixed this, but --dns-alias is not supported in 0.11. I'm quite sure we're still using the 0.10.1 in the evm ci